### PR TITLE
[MBL-18041][Student][Parent] Point based grading schemes

### PR DESCRIPTION
**Pull request by hermannakos**
Test plan: See ticket. Compare with web.

refs: MBL-18041
affects: Student, Parent
release note: Added support for point based grading schemes.

## Screenshots

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

## Checklist

- [ ] Follow-up e2e test ticket created or not needed
- [x] Run E2E test suite
- [x] Tested in dark mode
- [x] Tested in light mode
- [x] Approve from product

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by kristofnemere — APPROVED** [review]
QA+1

File: libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/OfflineDatabaseMigrationTest.kt [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.IOException
+
+private const val TEST_DB = "migration-test"
+private val ALL_MIGRATIONS = offlineDatabaseMigrations.takeLast(appDatabaseMigrations.size).toTypedArray()
+
+@RunWith(AndroidJUnit4::class)
+class AppDatabaseMigrationTest {
```
Comment (tamaskozmer) [inline libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/OfflineDatabaseMigrationTest.kt:33]: I think this class is a victim of some copypasta 😄

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

**Review by tamaskozmer — APPROVED** [review]

File: libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/OfflineDatabaseMigrationTest.kt (hunk 1 above)
Comment (hermannakos) [reply libs/pandautils/src/androidTest/java/com/instructure/pandautils/room/offline/OfflineDatabaseMigrationTest.kt:33]: oops 🫣

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Native Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Pull request merged**